python/kvikio/kvikio/benchmarks/utils.py

```diff
@@ -41,6 +42,10 @@ def pprint_sys_info() -> None:
         gpu_name = f"{pynvml.nvmlDeviceGetName(dev)} (dev #0)"
         mem_total = format_bytes(pynvml.nvmlDeviceGetMemoryInfo(dev).total)
         bar1_total = format_bytes(pynvml.nvmlDeviceGetBAR1MemoryInfo(dev).bar1Total)
+    if version == (0, 0):
+        libcufile_version = "unknown"
+    else:
+        libcufile_version = f"{version[0]}.{version[1]}"
```

Comment:
```suggestion
        libcufile_version = ".".join(version)
```

Reply:
Reverted back to `f"{version[0]}.{version[1]}"`, mypy complains because `version` is integers and not strings.